README.md

```diff
@@ -36,7 +36,8 @@ Bevy is still in the _very_ early stages of development. APIs can and will chang
 
 ## Community
 
-Before contributing or participating in discussions with the community, you should familiarize yourself with our **[Code of Conduct](https://github.com/bevyengine/bevy/blob/master/CODE_OF_CONDUCT.md)**
+Before contributing or participating in discussions with the community, you should familiarize yourself with our **[Code of Conduct](https://github.com/bevyengine/bevy/blob/master/CODE_OF_CONDUCT.md)** and 
+**[How to Contribute](https://github.com/bevyengine/bevy/blob/master/CONTRIBUTING.md)**
```

Comment:
This should probably be changed to the bevy book link

Reply:
Good point! Done.